appveyor.ini

```diff
@@ -0,0 +1,53 @@
+[General]
+Branch = master
+ShallowClone = True
+
+# Variables defined here override the default value
+# The variable names are casesensitive
+[Variables]
+#Values need to be overwritten to create a chache
+UseCache = True
+CreateCache = False
+
+# Settings applicable for all Crafts matrices
+# Settings are Category/key=value
+# Category is case sensitive
+[GeneralSettings]
+
+## This is the location of your python installation.
+## This value must be set.
+Paths/Python = C:\Python39-x64
+Paths/Python27 = C:\Python27-x64
+
+Compile/BuildType = RelWithDebInfo
+
+Compile/UseNinja = True
+
+Paths/downloaddir = ${Variables:Root}\downloads
+ShortPath/Enabled = False
+ShortPath/EnableJunctions = True
+ShortPath/JunctionDir = C:\CM-SP\
+
+; Packager/RepositoryUrl = https://files.kde.org/craft/
+Packager/PackageType = NullsoftInstallerPackager
+Packager/RepositoryUrl = http://ftp.acc.umu.se/mirror/kde.org/files/craft/master/
+
+ContinuousIntegration/Enabled = True
+
+## This option can be used to override the default make program
+## change the value to the path of the executable you want to use instead.
+Compile/MakeProgram = jom
+
+Packager/UseCache = ${Variables:UseCache}
+Packager/CreateCache = ${Variables:CreateCache}
+Packager/CacheDir = ${Variables:Root}\cache
+
+[BlueprintSettings]
+# don't try to pip install on the ci
+python-modules.ignored = True
+nextcloud-client.buildTests = True
+binary/mysql.useMariaDB = False
+
+[windows-msvc2019_64-cl]
+QtSDK/Compiler = msvc2019_64
+General/ABI = windows-msvc2019_64-cl
```

Comment:
These changes also seem unrelated, it looks like there has been a mistake somewhere, maybe rebasing the branch?